src/components/login/login.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import './login.css';
+import {Link, Redirect} from "react-router-dom";
+import axios from "axios";
+import Button from "../button";
+import Header from "../header";
+import Label from "../label";
+import Input from "../input";
+
+
+export default class Login extends Component{
+
+    constructor(props) {
+        super(props);
+    }
+
+    state = {
+        username: '',
+        password: '',
+    }
+
+    handleChange  = (event) => {
+        const {name, value} = event.target;
+        this.setState({
+            [name]: value
+        });
+    };
+
+    handleSubmit  = (event) => {
+        event.preventDefault();
+        const { username, password } = this.state
+        axios({
+            method: "POST",
+            data: {
+                username: username,
+                password: password,
+            },
+            withCredentials: true,
+            url: "http://localhost:3001/user/login",
+        })
+            //This is probably not the best way to authenticate a user. A check on the status code would be better.
+        .then(response => {
+            console.log(response.data)
+            if (!response.data.success) {
+                console.log(response.data)
+            } else {
+                //Need to store the user data in local storage
+                Object.keys(response.data.info).forEach((item) => {
+                    localStorage.setItem(item, response.data.info[item])
+                });
+                //This pushes you to the profile page on successful login.
+                const username = localStorage.getItem('username')
+                this.props.history.push('/profile/' + username);
```

Comment:
Removed all local storage stuff Ellie and I will change this to new method when we merge.